ebml/EbmlUnicodeString.h

```diff
@@ -55,9 +55,9 @@ class EBML_DLL_API UTFstring {
   using value_type = wchar_t;
 
   UTFstring() = default;
-  UTFstring(const wchar_t *); // should be NULL terminated
+  explicit UTFstring(const wchar_t *); // should be NULL terminated
   UTFstring(const UTFstring &);
-  UTFstring(std::wstring const &);
+  explicit UTFstring(std::wstring const &);
```

Comment:
These two particular changes are wrong, or rather, they're not what we want.

The UTFstring class is supposed to be a helper class. Things such as `auto elt = new EbmlUincodeString(L"something");` are supposed to work without explicit conversions, similar to how you'd define functions taking an `std::string const &` not requiring explicit casts when used with a `"just a string"`.

Yes, it actually does break the API, and MKVToolNix fails to compile, hence me noticing it.

I'll just revert these two things in a commit on top of your changes after merging.